docs/_snippets/portable-stories-import-annotations.md

```diff
@@ -1,11 +1,32 @@
-```js filename="setupFile.js|ts" renderer="common" language="js"
-// Adjust the import based on the supported framework or Storybook's testing libraries (e.g., react, testing-vue3)
-import { setProjectAnnotations } from '@storybook/your-framework';
+```js filename="setupFile.js|ts" renderer="react" language="ts"
+import { setProjectAnnotations } from '@storybook/react';
+import * as previewAnnotations from './.storybook/preview';
 
-// Replace this export with the default export from your Storybook preview file if you're working with a latest version of Storybook
-import * as projectAnnotations from './.storybook/preview';
+// Apply the global annotations from the Storybook preview file
+const annotations = setProjectAnnotations(previewAnnotations);
```

Comment:
Should we add the `testingLibraryRender` annotation here too?
```suggestion
const annotations = setProjectAnnotations([previewAnnotations, { testingLibraryRender }]);
```

Reply:
I actually replaced the usage of this snippet with the better one we already updated, but forgot to delete it.